install_ruby.sh

```diff
@@ -78,9 +79,20 @@ fi
   rm -rf /tmp/ruby-build
 )
 
-gem update --system "$RUBYGEMS_VERSION"
+case $RUBY_VERSION in
+  trunk)
+    # DO NOTHING
+    ;;
+  2.7.*)
+    # DO NOTHING
+    ;;
+  2.6.*)
+    # DO NOTHING
+    ;;
+  *)
+    gem update --system "$RUBYGEMS_VERSION"
+    gem install bundler --version "$BUNDLER_VERSION" --force
+    ;;
+esac
 
 rm -fr /usr/src/ruby /root/.gem/
-
-# rough smoke test
-ruby --version && gem --version && bundle --version
```

Comment:
I think that `(cd && ruby --version && gem --version && bundle --version)` will work.